apps/next/src/components/Header/AddressList.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Stack,
+  useTheme,
+  getHexAlpha,
+  CChainIcon,
+  XPChainIcon,
+  SolanaColorIcon,
+  EthereumColorIcon,
+  BitcoinColorIcon,
+  Grow,
+} from '@avalabs/k2-alpine';
+import { AddressItem } from '@/pages/AccountManagement/components/AddressItem';
```

Comment:
Let's move it out of `pages/AccountManagement` into a common space, e.g. `src/components/Address/AddressItem.tsx`.